.github/workflows/python.yml

```diff
@@ -49,7 +55,28 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: '3.9'
-      - name: Setup application
-        run: make setup
+      - name: setup mise
+        uses: jdx/mise-action@v2
+        with:
+          experimental: true
+      - name: Run dead code
+        run: mise run check:dead-code
+
+  publish_package:
```

Comment:
Include a "depends on" just to make sure.

Reply:
that's the `needs` field that is added (line 68)